### ISSUE TITLE
cant execute nhl setup

### ISSUE THREAD

**Issue by ceegmac** [mikemountain/nfl-led-scoreboard#19]
Hi, just struggling getting my scoreboard setup after the updates.
  _______________  _____
  / __/ __/_  __/ / / / _ \
 _\ \/ _/  / / / /_/ / ___/
/___/___/ /_/  \____/_/

1.5.0
Traceback (most recent call last):
  File "nhl_setup.py", line 1605, in <module>
  File "nhl_setup.py", line 1518, in main
PermissionError: [Errno 13] Permission denied: 'config/.default/firstrun'
[1130] Failed to execute script nhl_setup
pi@raspberrypi:~/nhl-led-scoreboard $

basically that is what i am getting when i try to run setup. 
had it working before and i dont think my up date went well. and scoreboard boots it doesnt do anything. can get the test screen  "HELLO WORLD" to work though

any help is appreciated

thank you

**Comment by mikemountain** [comment]
Think you have the wrong repo, this is the N**F**L scoreboard repo

**Comment by ceegmac** [comment]
sorry about that.

But I am planning on making the NFL one before season. cant wait. thank you
for doing that

**Comment by mikemountain** [comment]
No problem, just wanted to confirm. Enjoy the season, and go leafs go woo

**Comment by siebe41** [comment]
Fwiw make sure you mark the file as executable with chmod

**Comment by ceegmac** [comment]
sorry to be daft but how do you do that

**Comment by siebe41** [comment]
chmod +x yourfilename
